spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfigurationTests.java

```diff
@@ -41,7 +41,8 @@
 	private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 			.withConfiguration(AutoConfigurations.of(GcpDatastoreAutoConfiguration.class,
 					GcpContextAutoConfiguration.class, DatastoreTransactionManagerAutoConfiguration.class,
-					DatastoreRepositoriesAutoConfiguration.class))
+					DatastoreRepositoriesAutoConfiguration.class,
+					TestRepository.class))
```

Comment:
Why did you add `TestRepository` here? That circumvents the test below which is supposed to create the TestRepository instance using the Spring Data Datastore Repo factory instead of pulling a bean from the context....

Reply:
I observed no bean definition found exception while running mvn clean install with out TestRepository.class

Reply:
That means this PR is faulty and it's a problem that needs to be fixed because this repository class was intentionally NOT added and it worked before.